etc/omero.properties

```diff
@@ -911,7 +912,7 @@ versions.log4j=1.2.17
 versions.logback=1.1.1
 ## Change url in antlib/resources/release.xml when changing lucene version
 versions.lucene=2.4.1
-versions.mail=1.4.1
+versions.mail=1.4.4
```

Comment:
This bump may have to be double-checked by the mail crew /cc @aleksandra-tarkowska @joshmoore